### PR TITLE
add: Functionality to generate a ClusterConfiguration Template in the next notebook cell

### PR DESCRIPTION
**Pull request by VanillaSpoon**
# Issue link

# What changes have been made
This PR adds a new util function, generateClusterConfig. 

The function will simplify the process of setting up a cluster by providing an unfilled ClusterConfiguration in the following cell.

# Verification steps
- Build the sdk
- Open a new Notebook and install the sdk
- Import the generateClusterConfig function: 
```
from codeflare_sdk import Cluster, ClusterConfiguration, TokenAuthentication, generateClusterConfig
```

- Now, running `generateClusterConfig()` will add a new cell in the notebook with a ClusterConfiguration template:
```
cluster = Cluster(ClusterConfiguration(
    name='',
    head_cpus='',
    head_memory='',
    head_gpus='', # For GPU enabled workloads set the head_gpus and num_gpus
    num_gpus='',
    num_workers='',
    min_cpus='',
    max_cpus='',
    min_memory='',
    max_memory='',
    image="", # Optional Field
    write_to_file=False, # When enabled Ray Cluster yaml files are written to /HOME/.codeflare/resources
    local_queue="" # Specify the local queue manually
))
```

## Checks
- [ ] I've made sure the tests are passing. 
- Testing Strategy
   - [ ] Unit tests
   - [ ] Manual tests
   - [ ] Testing is not required for this change

### CODE REVIEW

**Review by ChristianZaccaria — COMMENTED** [review]
Thanks for the idea Eoin! I was wondering about this myself, but found this really cool keyboard command which will show all function arguments plus detailed description about them.

TLDR: after importing, after the `(` do SHIFT+TAB.

example:
image
